vdr/cmd/cmd.go

```diff
@@ -117,7 +115,9 @@ func updateCmd() *cobra.Command {
 }
 
 func resolveCmd() *cobra.Command {
-	return &cobra.Command{
+	var printMetadata bool
```

Comment:
requires rebase on master after #81 has been merged

Reply:
Rebasing master is my speciality